### PR TITLE
solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**

### CODE REVIEW

File: alembic/README
Comment (diana-shyrokikh) [inline alembic/README:1]: Add a blank line)

File: main.py
Comment (diana-shyrokikh) [inline main.py:1]: Same here)

**Review by diana-shyrokikh — APPROVED** [review]

File: README.md
Comment (LiudmylaKulinchenko) [inline README.md:1]: You can remove task description

File: city_api/city_router.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+async def create_city(new_city: schemas.CityCreate, db: AsyncSession = Depends(get_db)):
+    return await crud.create_city(db, new_city)
+
+
+@router.get("/cities/", response_model=list[schemas.City])
+async def get_all_cities(db: AsyncSession = Depends(get_db)):
+    return await crud.get_cities(db)
+
+
+@router.get("/cities/{city_id}", response_model=schemas.City)
```
Comment (LiudmylaKulinchenko) [inline city_api/city_router.py]: missed /

File: city_api/city_router.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    city = await crud.get_city(db, city_id)
+    if city is None:
+        raise HTTPException(
+            status_code=400,
+            detail="There is no such city"
+        )
+    return city
+
+
+@router.delete("/cities/{city_id}", response_model=dict)
```
Comment (LiudmylaKulinchenko) [inline city_api/city_router.py]: /

File: temperature_api/crud.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+async def update_temperatures(db: AsyncSession):
+    async with db.begin():
+        cities = await db.execute(select(city_api.models.City))
+
+        for city in cities.scalars():
+            # Get the current temperature for each city
+            temperature = await get_temperature_from_weatherapi(city.name)
+
+            # Create a new Temperature record
```
Comment (LiudmylaKulinchenko) [inline temperature_api/crud.py]: remove comments

File: temperature_api/temperature_router.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+router = APIRouter()
+
+
+@router.get("/temperatures/", response_model=list[schemas.Temperature])
+async def get_all_temperatures(db: AsyncSession = Depends(get_db)):
+    return await crud.get_temperatures(db)
+
+
+@router.get("/temperatures/{temp_id}", response_model=schemas.Temperature)
```
Comment (LiudmylaKulinchenko) [inline temperature_api/temperature_router.py]: /

File: temperature_api/temperature_router.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    temp = await crud.get_temperature(db, temp_id)
+    if temp is None:
+        raise HTTPException(
+            status_code=400,
+            detail="There is no such temperature"
+        )
+    return temp
+
+
+@router.post("/temperatures/update")
```
Comment (LiudmylaKulinchenko) [inline temperature_api/temperature_router.py]: /

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done 👏

**Pull request closed without merging**